src/settings/command_line.rs

```diff
@@ -11,6 +11,8 @@ use crate::error::{usage_error, ShellError};
 #[derive(Debug, StructOpt, Default)]
 #[structopt(name = "qldb", about = "A shell for interacting with Amazon QLDB.")]
 pub struct Opt {
+    /// The AWS Region code of the QLDB ledger to connect to. For example: `us-east-1`.
+    /// If not provided, the shell connects to your default AWS Region as specified in your AWS profile.
```

Comment:
I believe we use the `AWS_DEFAULT_REGION` and `AWS_REGION` environment variables too.

I don't think we should explain all defaults here - that is best left to documentation. My suggestion is you change the second sentence to "By default, the shell will pick a default region as described in the documentation." (or similar)

Reply:
Updated